workflows/prognostic_c48_run/docs/config-usage.rst

```diff
@@ -167,5 +167,15 @@ Diagnostics to be output by the Fortran model are specified in the
 to the Fortran ``diag_table`` representation of diagnostics (see fv3config_ docs).
 
 
+Chunking
+^^^^^^^^
+
+The desired chunking can be specified for each diagnostic file to be output. For
```

Comment:
Usage docs are great, but more likely to become out of date since someone editing the code won't see them.

It doesn't look like this is mentioned in the reference API. Please update the docstrings of FortranFileConfig and DiagnosticsFileConfig, and defer as much of this discussion as possible to the reference APIs.

Reply:
Hmm. I don't think the FortranFileConfig etc should know about requirements for the segmented run, which is the main point I'm trying to make here. Ideally, at some point there might be a "segmented run" class which could have this info in its reference API, but for now I think it needs to be in the usage docs.

Reply:
Ok. I agree this is scoped to the segmented run rather than the runtime requirements. It just feels off to have these specific details listed in this general config usage section, while leaving it undocumented here: https://github.com/VulcanClimateModeling/fv3net/blob/29a3fa839b8ee90f9dc94bc95b22cde6d7796704/workflows/prognostic_c48_run/docs/local-execution.rst#segmented-runs.
 
You already have a cross reference to `segmented-run-cli`. Can you move these details there, and just provide a cross reference without repeating the info? 
```suggestion
The desired chunking can be specified for each diagnostic file to be output. For
.. note :: Not all chunks are valid for segmented runs. See :ref:`segmented-run-cli` for more details.
```